.appveyor.yml

```diff
@@ -20,9 +20,12 @@ environment:
   VisualStudioVersion: 15.0
   TreatWarningsAsErrors: true
   codecov_token: b8be69b5-6dc3-40f8-a4c7-4e13a42654a3
-build_script:
-- dotnet build src
-- dotnet pack src --no-build
+build_script: |
+  dotnet build src
+  dotnet pack src --no-build
+  yarn install --cwd src/nerdbank-streams
+  .\src\nerdbank-streams\node_modules\.bin\tsc -p src\nerdbank-streams\gulpfile.tsconfig.json
```

Comment:
> seems odd to have to invoke TSC separately from gulp, if you're going to use gulp

This tsc invocation is what produces the `gulpfile.js` file. I only check in the `gulpfile.ts` file, so part of the build has to be to compile that to `gulpfile.js`. :smile:

> Also, would expect that one of your gulp tasks would simply be this build

In fact gulp *does* run tsc (with `tsconfig.json` instead of `gulpfile.tsconfig.json`) in order to compile the actual project.

> which would also do the yarn install, then the TSC, then any packaging you need

gulp can't run the yarn install because gulp isn't present yet. I use yarn to get gulp. But yes, gulp does build the package.